content/4_Gold/Knapsack_DP.problems.json

```
@@ -72,8 +72,7 @@
       "isStarred": false,
       "tags": ["Knapsack"],
       "solutionMetadata": {
-        "kind": "link",
-        "url": "https://codeforces.com/blog/entry/70018"
+        "kind": "internal",
```

```suggestion
        "kind": "internal"
```
should fix the build